libs/design-system/src/lib/Pages/GettingStartedPage/index.tsx

```diff
@@ -0,0 +1,435 @@
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+
+import { LandingHeader } from '../../Components/LandingHeader';
+import { LinkCard2Column } from '../../Components/LinkCard2Column';
+import { ReactComponent as CadenceIcon } from '../../../../images/tools/tool-cadence.svg';
+import { ReactComponent as FCLIcon } from '../../../../images/tools/tool-fcl.svg';
+import { SocialLinksSignup } from '../../Components';
+import { SDKCard } from '../../Components/SDKCard';
+import { Button } from '../../Components/Button';
+import FeaturedArticleCard from '../../Components/FeaturedArticleCard';
+import { ToolCard } from '../../Components/ToolCard';
+import { ContentNavigation } from '../../Components/ContentNavigation';
+import { LinkCard3Column } from '../../Components/LinkCard3Column';
+import { ReactComponent as GitHubIcon } from '../../../../images/social/github.svg';
+import { ReactComponent as DiscordIcon } from '../../../../images/social/discord.svg';
+
+const svgToDataUri = (element: Parameters<typeof renderToStaticMarkup>[0]) => {
```

Comment:
yeah it might be a good idea to create a `utils` or `common` folder or something as we get more shared functionality but I'll import it from `LinkCard2Column.stories` for now